cypress/utils/utils.ts

```diff
@@ -73,12 +73,12 @@ export function exists(value: string): void {
 
 export function notExists(value: string): void {
     // Wait for DOM to render table and sibling elements
-    selectItemsPerPage(100);
     cy.wait(2000);
     cy.get(commonView.appTable)
         .next()
         .then(($div) => {
             if (!$div.hasClass("pf-c-empty-state")) {
+                selectItemsPerPage(100);
```

Comment:
I think previous addition might not be working, as that nav button is disabled when page is empty. 
Anyways, I have covered that in pagination PR. Old code would also work now. 
Let's keep it, no harm in having this call here.